docs/source/guides/deploy_solver_new.rst

```diff
@@ -105,18 +128,75 @@ Schema property
 
 The schema of an application is the configuration used to solve a problem. This schema needs to have at least the `timeLimit` and the `solver` properties. Besides that, it's up to the developer to decide which configuration is needed. If you want to know how to define a json-schema, check the section :ref:`Write a json-schema`.
 
+Conventions
+*****************
+
+We follow some common conventions in the configuration schema so most apps share most of the main properties. Below is a list of known properties and what the usually represent.
+
+#. **timeLimit**: float. It indicates the amount of seconds before the method should stop.
```

Comment:
This may be confusing, because when using a mathematical solver (cbc or cplex, for example) we use that as the time limit for the mathematical solver, not the solver itself, so the time it takes to solve the problem could be bigger based on the time needed to process the data and the solution.

I do not know if this merits a mention or not.

Reply:
if the pre-processing is long enough to do something, then whoever uses that should substract the time it takes to pre-process before passing the parameter to the mathematical solver. It could merit a mention but I'm not sure where. We do not mention mathematical solvers here so I don't think it makes sense to include the term.